core/src/main/java/org/apache/accumulo/core/file/blockfile/impl/CachableBlockFile.java

```diff
@@ -514,96 +479,40 @@ private CachedBlockRead(SeekableByteArrayInputStream seekableInput, CacheEntry c
       super(seekableInput);
       this.seekableInput = seekableInput;
       this.cb = cb;
+      indexable = true;
+    }
+
+    /**
+     * It is intended that the caller of this method will close the stream we also only intend that
+     * this be called once per BlockRead. This method is provide for methods up stream that expect
+     * to receive a DataInputStream object.
+     */
+    public DataInputStream getStream() {
```

Comment:
It seems like this method is no longer needed.

Reply:
This could be another refactoring opportunity but currently its used by an RFile call to getMetaStore(), which returns a DataInputStream.